api/validator_test.go

```diff
@@ -12,25 +12,81 @@ import (
+}
+
+type bankNumberExpectedParameter struct {
+	code    int
+	message string
+}
+
+var validationBanksAcceptedRulesAndFees = []bankNumberParameter{
+	{input: models.Caixa, expected: true},
+	{input: models.Stone, expected: true},
+	{input: models.BancoDoBrasil, expected: false},
+	{input: models.Santander, expected: false},
+	{input: models.Citibank, expected: false},
+	{input: models.Itau, expected: false},
+	{input: models.JPMorgan, expected: false},
+	{input: models.Pefisa, expected: false},
+}
+
+var validationRulesAndFeesSuccessParametersV2 = []bankNumberParameter{
+	{input: models.Caixa, expected: bankNumberExpectedParameter{code: 200}},
+	{input: models.Stone, expected: bankNumberExpectedParameter{code: 200}},
+}
+
+var validationFeesFailedParametersV2 = []bankNumberParameter{
+	{input: models.BancoDoBrasil, expected: bankNumberExpectedParameter{code: 400, message: `{"errors":[{"code":"MP400","message":"title.fees not available for this bank"}]}`}},
+	{input: models.Santander, expected: bankNumberExpectedParameter{code: 400, message: `{"errors":[{"code":"MP400","message":"title.fees not available for this bank"}]}`}},
+	{input: models.Citibank, expected: bankNumberExpectedParameter{code: 400, message: `{"errors":[{"code":"MP400","message":"title.fees not available for this bank"}]}`}},
+	{input: models.Itau, expected: bankNumberExpectedParameter{code: 400, message: `{"errors":[{"code":"MP400","message":"title.fees not available for this bank"}]}`}},
+	{input: models.JPMorgan, expected: bankNumberExpectedParameter{code: 400, message: `{"errors":[{"code":"MP400","message":"title.fees not available for this bank"}]}`}},
+	{input: models.Pefisa, expected: bankNumberExpectedParameter{code: 400, message: `{"errors":[{"code":"MP400","message":"title.fees not available for this bank"}]}`}},
+}
+
+var validationRulesFailedParametersV2 = []bankNumberParameter{
+	{input: models.BancoDoBrasil, expected: bankNumberExpectedParameter{code: 400, message: `{"errors":[{"code":"MP400","message":"title.rules not available for this bank"}]}`}},
+	{input: models.Santander, expected: bankNumberExpectedParameter{code: 400, message: `{"errors":[{"code":"MP400","message":"title.rules not available for this bank"}]}`}},
+	{input: models.Citibank, expected: bankNumberExpectedParameter{code: 400, message: `{"errors":[{"code":"MP400","message":"title.rules not available for this bank"}]}`}},
+	{input: models.Itau, expected: bankNumberExpectedParameter{code: 400, message: `{"errors":[{"code":"MP400","message":"title.rules not available for this bank"}]}`}},
+	{input: models.JPMorgan, expected: bankNumberExpectedParameter{code: 400, message: `{"errors":[{"code":"MP400","message":"title.rules not available for this bank"}]}`}},
+	{input: models.Pefisa, expected: bankNumberExpectedParameter{code: 400, message: `{"errors":[{"code":"MP400","message":"title.rules not available for this bank"}]}`}},
+}
+
+var validationRulesSuccessParametersV1 = []bankNumberParameter{
```

Comment:
@davidsmedeiros Aqui pelo que eu entendi do teste é a validando a rota V1 nos casos de sucesso, logo não tem Rules.. Eu tiraria o nome Rules daqui pq acho q ele acaba confundindo.

```suggestion
var validationSuccessParametersV1 = []bankNumberParameter{
```

Reply:
done